docs/node/workflows.md

```diff
@@ -65,54 +65,81 @@ There is no official support for querying the state of asynchronously started Wo
 
 ## How to cancel a Workflow Execution
 
-In the Node SDK, Workflows are represented internally by a tree of scopes. The main function runs in the root scope.
-Cancellation propagates from outer scopes to inner ones and is handled by catching `CancelledError`s
-thrown by cancellable operations (see below).
+In the Node SDK, Workflows are represented internally by a tree of **Cancellation Scopes**, each with cancellation behaviors you can specify.
+The Workflow `main` function is the only scope with no parent, running as the "root" scope.
```

Comment:
Everything runs in the root scope unless you explicitly create a new scope.

Reply:
```suggestion
Everything runs in the "root" scope by default.
```